src/azure-cli-core/tests/test_progress.py

```diff
@@ -0,0 +1,153 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import unittest
+import mock
+
+import azure.cli.core.commands.progress as progress
```

Comment:
`from azure.cli.core.commands.progress import (ProgressType, etc...)`
Then you don't need to keep doing `progress.X` in code.